modules/social_features/social_event/src/Form/EventSettingsForm.php

```diff
@@ -78,7 +87,11 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
 
     $this->configFactory->getEditable('social_event.settings')
       ->set('enroll', $group_type_ids)
+      ->set('address_visibility_settings', $form_state->getValue('address_visibility_settings'))
       ->save();
+
+    // Invalidate cache tags to refresh blocks of list of events.
+    \Drupal::service('cache_tags.invalidator')->invalidateTags(['node_list']);
```

Comment:
@denysyukm Good adding here. But are you sure that node_list tag will be enough? What about event detail page or featured events on landing pages?

Reply:
I checked a lot and at the moment I didn't find any cache issues.